tests/dragonfly/cluster_test.py

```diff
@@ -835,5 +835,128 @@ async def test_cluster_slot_migration(df_local_factory: DflyInstanceFactory):
     except redis.exceptions.ResponseError as e:
         assert e.args[0] == "Can't start the migration, another one is in progress"
 
+    await push_config(
+        config.replace("LAST_SLOT_CUTOFF", "5259").replace("NEXT_SLOT_CUTOFF", "5260"),
+        c_nodes_admin,
+    )
+
+    await c_nodes_admin[0].close()
+    await c_nodes_admin[1].close()
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_cluster_data_migration(df_local_factory: DflyInstanceFactory):
```

Comment:
Do we need 2 tests for data and slot migration? Why not do both in the same test? What's the benefit of having 2 tests?

Reply:
I don't want to have a really big test. in cluster_migration_test I check corner cases of API and in current test data.